pkg/skaffold/kubernetes/logger/log.go

```diff
@@ -152,6 +153,11 @@ func (a *LogAggregator) Stop() {
 	close(a.events)
 }
 
+func (a *LogAggregator) isNil() bool {
+       // This instance may be from an interface
```

Comment:
```
// This instance may be from an interface
```

what does this mean? I'd get it if `LogAggregator` was an interface but I'm totally lost on this one.